apps/gallery/index.html

```diff
@@ -20,6 +20,7 @@
     <meta name="availableLanguages" content="ar, en-US, fr, zh-TW">
     <link rel="localization" href="locales/gallery.{locale}.properties">
     <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
+    <link rel="localization" href="/shared/locales/media/media.{locale}.properties">
```

Comment:
I think you'll need to do this for Camera as well, since it also uses media_frame and video_player.